webpack.config.js

```diff
@@ -75,7 +75,15 @@ module.exports = (env, argv) => {
                         },
                     ],
                 },
+                {
+                    test: /\.(png|woff|woff2|eot|ttf|svg)(\?v=[0-9]\.[0-9]\.[0-9])?$/,
+                    use:[
+                        {
+                            loader: 'url-loader'
+                        }
```

Comment:
Presumably this is for the Katex fonts... where is this documented? what does this do to the bundle size?

Reply:
Correct @alexcjohnson 👍 I would guess this is not the optimal / final way of including KaTeX fonts wrt. bundle size and loading speed, if KaTeX support in `dcc.Markdown` is something worth adding. Maybe keep KaTeX as an external thing 📦 (like [`highlight.js` is already](https://github.com/plotly/dash-core-components/blob/7bdeec55ea8734cc08fcc21086acad797b617238/dash_core_components/__init__.py#L50)) and load [KaTeX globally](https://katex.org/docs/browser.html#loading-as-global)?

_Another thing to consider if including math in the markdown component:_ Should it have an argument stating if math is to be rendered or not (such that it is not a breaking change if e.g. the user writes `$some text which should not be inline math$`)? Or maybe a better alternative: do not use `$...$` for inline math, but rather e.g. the [MathJax default `\(...\)`](http://docs.mathjax.org/en/latest/basic/mathematics.html?highlight=inline#tex-and-latex-input).

Reply:
True, it wouldn't be hard to end up with two `$` on a line together accidentally. Thanks for bringing this up! So yeah, let's make this opt-in. Perhaps also with an option to choose new delimiters, though that's not so important if math rendering is opt-in.

Global loading and fonts frankly sounds like a headache - particularly as we need to maintain the ability to run airgapped. Perhaps the best would be to hold this PR until we finish sorting out lazy loading #616, then make sure KaTeX is only loaded when rendering math-enabled markdown? Thoughts @Marc-Andre-Rivet?